doc/source/management/ssl_compatibility.rst

```diff
@@ -0,0 +1,60 @@
+SSL improvements
+================
+
+|PS| have undergone several SSL compatibility improvements, distinguishing it
+from the original MySQL commuinty edition. 
+
+Linking with OpenSSL
+--------------------
+
+As stated in `<https://dev.mysql.com/doc/refman/5.5/en/openssl-versus-yassl.html>`_,
+MySQL can be compiled using either OpenSSL toolkit or yaSSL embedded library to
+enable encrypted connections based on the OpenSSL API. All binary packages of
```

Comment:
should be discussing network protocols, not APIs: encrypted connections based on Transport Layer Security / Secure Sockets Layer